src/components/views/messages/MImageBody.js

```diff
@@ -152,8 +134,7 @@ module.exports = React.createClass({
                         <img className="mx_MImageBody_thumbnail" src={thumbUrl} ref="image"
                             alt={content.body}
                             onMouseEnter={this.onImageEnter}
-                            onMouseLeave={this.onImageLeave}
-                            onLoad={this.props.onImageLoad} />
```

Comment:
I believe this isn't needed as we give images explicit sizes again, 
since https://github.com/matrix-org/matrix-react-sdk/pull/248, so don't 
need to fire an onLoad any more.